src/mapgen/mg_decoration.h

```diff
@@ -72,7 +72,7 @@ class Decoration : public ObjDef, public NodeResolver {
 	s16 nspawnby;
 	s16 place_offset_y = 0;
 
-	std::unordered_set<u8> biomes;
+	std::unordered_set<u16> biomes;
```

Comment:
`biome_t` ?

Reply:
Ok i assumed 'biome_t' would not work here, but it probably does. Ok will do.

Reply:
Done here and in one other place.